src/common/plugins.js

```diff
@@ -107,19 +108,22 @@ class Plugins extends EventEmitter {
     try {
       this.reset()
       this.config = await load(this.configFile)
+      this.config.forEach(config => {
+        config.enabled = !(config.enabled === false)
+      })
     } catch (error) {
       if (error.code !== 'ENOENT') {
         warn(`failed to load plugin config: ${error.message}`)
       } else {
         if (autosave) await this.save()
       }
     }
-
     return this
   }
 
-  async reloadAndCreate() {
+  async reloadScanCreate() {
```

Comment:
I can't think of a case where we need reload but not scan, so shall we:
* move scan into reload
* get rid of reloadAndScan
* rename `reloadScanCreate` into something sane and call that from window

Reply:
Sounds good. Arguably we don't actually need `reloadAndCreate` we can just call reload and then create in window.